back_end/src/main/java/com/software/grey/services/implementations/SavedPostServiceImpl.java

```diff
@@ -70,17 +65,19 @@ public Page<PostDTO> getSavedPosts(PostFilterDTO postFilterDTO) {
         String userName = securityUtils.getCurrentUserName();
         List<String> feelings = Optional.ofNullable(postFilterDTO.getFeelings())
                 .filter(list -> !list.isEmpty())
-                .map(list -> list.stream().map(Enum::name).collect(Collectors.toList()))
-                .orElseGet(() -> Arrays.stream(Feeling.values()).map(Enum::name).collect(Collectors.toList()));
+                .map(list -> list.stream().map(Enum::name).toList())
+                .orElseGet(() -> Arrays.stream(Feeling.values()).map(Enum::name).toList());
 
         Pageable pageable = PageRequest.of(
                 postFilterDTO.getPageNumber(),
                 postFilterDTO.getPageSize());
 
         return savedPostRepository.findSavedPostsByUsernameAndDayMonthYear(userName, feelings,
-                postFilterDTO.getDay(),
-                postFilterDTO.getMonth(),
-                postFilterDTO.getYear(), pageable).map(SavedPost::getPost)
+                        postFilterDTO.getDay(),
+                        postFilterDTO.getMonth(),
+                        postFilterDTO.getYear(),
```

Comment:
later we can make this one variable date as MySQL also has CURRENT_DATE variable